packages/unmock-core/src/generator.ts

```diff
@@ -32,6 +32,7 @@ function firstOrRandomOrUndefined<T>(arr: T[]): T | undefined {
     ? arr[0]
     : arr[Math.floor(Math.random() * arr.length)];
 }
+export const USE_EXPERIMENTAL_GENERATOR = { yes: false };
```

Comment:
Maybe allow turning it on by an environmental variable, it might be easier to test than with in-code booleans?

Reply:
I'll leave it as this for now, just because otherwise if we flipped the switch and used `state` it may wreak havoc. As the flag will be taken out soon-ish, I'm comfortable keeping it this way.